metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "theforeman-foreman",
-  "version": "15.0.0",
+  "version": "16.0.0",
```

Comment:
It was already bumped to 15, so this isn't needed. I noticed this in the odd 14.0.1..16.0.0 jump